scss/inc/_forms.scss

```
@@ -342,6 +342,12 @@ input[type="password"].viewable-hiddenbox-input {
         }
     }
 
+    &.error {
+        .select2-choices {
+            border-color: $error;
+        }
```

```suggestion
        .select2-container {
            > :first-child {
                border-color: $error;
            }
        }
```

This should cover most of the `select2` rendering options. It may have not only the multiple, but a single choice in the future.